websites/E/EternalNetwork TM/presence.ts

```diff
@@ -79,22 +79,22 @@ presence.on("UpdateData", async () => {
 			if (pathname.startsWith("/")) presenceData.state = ttl;
 
 			if (pathname.includes("/memberlist.php"))
-				presenceData.state = "Sneaking into member list !";
+				presenceData.state = "Viewing Forum Members";
 
 			if (search.includes("?mode=team"))
-				presenceData.state = "Checking out team list !";
+				presenceData.state = "Viewing Forum Staff";
 
 			if (pathname.includes("/partner"))
-				presenceData.state = "Cheking our partners !";
+				presenceData.state = "Viewing Forum Partners";
 
 			if (pathname.includes("/donation"))
-				presenceData.state = "Trying to make donation for the forum !";
+				presenceData.state = "Donation to the forum";
```

Comment:
+This is wrong

Reply:
The message is changed and now is not the correct one, because the reviewed page is to make a donation not to check available donations from other members.